pilot/pkg/model/push_context.go

```diff
@@ -985,6 +1002,32 @@ func (ps *PushContext) initDestinationRules(env *Environment) error {
 	return nil
 }
 
+// AuthPolicyForProxy returns the matching auth policy for a given proxy
```

Comment:
This should have the same name as the main API, i.e. AuthenticationPolicyForWorkload

Reply:
done